config/environments/development.rb

```diff
@@ -38,4 +38,13 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.paperclip_defaults = {
+    :storage => :s3,
+    :s3_credentials => {
+      :bucket => ENV['mythicaleatures'],
+      :access_key_id => ENV['AKIAJXIU2NTHSKIVAN5Q'],
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.